src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -3460,7 +3460,9 @@ protected DLNAThumbnailInputStream getGenericThumbnailInputStreamInternal(String
 		if (thumb != null && isCodeValid(this)) {
 			// A local file
 			if (new File(thumb).exists()) {
-				return DLNAThumbnailInputStream.toThumbnailInputStream(new FileInputStream(thumb));
+				try (FileInputStream fis = new FileInputStream(thumb)) {
+					return DLNAThumbnailInputStream.toThumbnailInputStream(fis);
```

Comment:
I am unsure about this one. Will DLNAThumbnailInputStream.toThumbnailInputStream still get what it wants after this change?